__tests__/app.test.js

```diff
@@ -124,6 +124,51 @@ describe("GET /api/articles", () => {
       })
     })
   })
+  test('200: returns an array of all articles sorted by date DESCENDING and not to have body property', () => {
```

Comment:
do you need to specify every time that it should not have a body property, or is it enough to say it in one test?